src/services/eventManager.js

```diff
@@ -0,0 +1,96 @@
+(function () {
```

Comment:
I am planning to add the dispatches for the the ui-leaflet-label plugin to listen for, once i've pushed that it will be a good example.